connector/datadogconnector/examples/config.yaml

```diff
@@ -74,6 +74,12 @@ connectors:
       #
       trace_buffer: 500
 
+      ## @param peer_tags - [BETA] Optional list of supplementary peer tags that go beyond the defaults. The Datadog backend validates all tags
+      ## and will drop ones that are unapproved. The default set of peer tags can be found at
```

Comment:
To make sure I understand this: This means that, by default, the tags on the list linked on this comment will be approved, and any additional tags added here will also be approved. Is that right?

Is there any way we can do some (soft) validation about this? i.e. if a payload has an unapproved tag, warn them about it?

Reply:
No, the default ones are always approved, additional ones may or may not be approved by backend. I don't think we can add any validation in the collector, as the backend controls what tags can be ingested and the validation rules are always subject to change  (this is similar to backend allowlist).